App/Components/KeyboardResponsiveContainer.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+import { Keyboard, EmitterSubscription, View, ViewStyle, LayoutAnimation } from 'react-native'
+
+interface Props {
+  style?: ViewStyle
+}
+
+interface State {
+  height: number
+}
+
+export default class KeyboardResponsiveContainer extends React.Component<Props, State> {
```

Comment:
We can use this anywhere we want the container view to resize with the keyboard.

Reply:
very cool